pkg/option/endpoint.go

```diff
@@ -9,6 +9,7 @@ var (
 		ConntrackLocal:      &specConntrackLocal,
 		Debug:               &specDebug,
 		DebugLB:             &specDebugLB,
+		XDPTracing:          &specXDPTracing,
```

Comment:
I think we don't need this option for endpoints, as there is no XDP prog used by endpoints.